index.js

```diff
@@ -0,0 +1,217 @@
+//. # Sanctuary Benchmark
+//.
+//. Small wrapper over [benchmarkjs][1] to enable a consistent style of
+//. benchmarking across all Sanctuary projects. Allows for the easy definition
+//. of comparative benchmarks, and outputs results in a standardized format for
+//. sharing.
+//.
+//. ## Usage
+//.
+//. ```js
+//. //
+//. // old-vs-new.js
+//. //
+//.
+//. //Imagine these are libs. Normally they would be require()'d.
+//. const oldVersion = (f, xs) => xs.map(f);
+//. const newVersion = (f, xs) => {
+//.   const len = xs.length; out = new Array(len);
+//.   for(let idx = 0; idx < len; idx++) out[idx] = f(xs[idx]);
+//.   return out;
+//. };
+//.
+//. const small = Array.from({length: 1}, (_, i) => i);
+//. const large = Array.from({length: 1000}, (_, i) => i);
+//.
+//. //Normally we wouldn't assign this to a var, but just export it.
+//. const benchmark = require('sanctuary-benchmark')(oldVersion, newVersion, {
+//.   'map/small': map => map(x => x + 1, small),
+//.   'map/large': map => map(x => x + 1, large)
```

Comment:
Nit: I like to use trailing commas when writing multiline object literals in an ES6 context.